### PR TITLE
Role adding when a ticket gets opened

### PR DESCRIPTION
**Pull request by DuckMasterAl**
Closes: Issue #28 
You might want to change the reason for adding the role.

### CODE REVIEW

**Comment by TheMoksej** [comment]
I, Moksej, approve this pull request.

**Comment by DuckMasterAl** [comment]
@Mergifyio refresh

**Comment by advaith1** [comment]
can we possibly also put the role ids in settings? cc @Cairo2k18 
i don't want to have any hardcoded ids

**Comment by DuckMasterAl** [comment]
> i don't want to have any hardcoded ids

Then you should do the same with the bot power role...

**Comment by advaith1** [comment]
yes we should

**Comment by DuckMasterAl** [comment]
@advaith1 Done, Ice said he'd update it.

File: cogs/tickets.py [hunk 1]
```diff
@@ -105,6 +105,12 @@ async def open_ticket(self, ctx, bot: discord.User):
             embed2.add_field(name="Developer", value=f"[{str(owner)}]({self.bot.settings['website']['url']}/users/"
                                                      f"{owner.id}) (`{owner.id}`)")
 
+            testguild = self.bot.get_guild(int(self.bot.settings['guilds']['staff']))
+            fixesrole = delguild.get_role(int(self.bot.settings['roles']['fixesBot']))
+            memberbot = delguild.get_member(bot.id)
```
Comment (advaith1) [inline cogs/tickets.py:110]:
```suggestion
            fixesrole = testguild.get_role(int(self.bot.settings['roles']['fixesBot']))
            memberbot = testguild.get_member(bot.id)
```

**Comment by DuckMasterAl** [comment]
Someone link this pr to issue #28

**Comment by advaith1** [comment]
@DuckMasterAl u should have just put "closes #28" in the desc

**Comment by advaith1** [comment]
@IceeMC did u update the settings file??

**Comment by DuckMasterAl** [comment]
I did just in a fancier way with colons

**Comment by advaith1** [comment]
if u do it fancy then it wont work

**Comment by IceeMC** [comment]
Yes, I told you I did advaith

**Comment by DuckMasterAl** [comment]
> if u do it fancy then it wont work


Yea I know, it's ok I can just close it.

**Comment by Cairo2k18** [comment]
ree I need to refactor this later

**Pull request merged**